Formula/freerdp.rb

```diff
@@ -25,7 +25,6 @@ class Freerdp < Formula
   on_linux do
     depends_on "ffmpeg"
     depends_on "glib"
-    depends_on "systemd"
```

Comment:
@iMichka Note that this will need removed because it doesn't exist in homebrew-core.